news/2023-01-01-osumonthly-6.md

```diff
@@ -0,0 +1,331 @@
+
+Thanks to [Kurboh](https://osu.ppy.sh/users/15243233) for providing this cool fact, you can view the sources for the fact here: [1](https://osu.ppy.sh/community/forums/topics/43115?n=12), [2](/wiki/shared/news/2023-01-01-osumonthly-6/didyouknow-dec22.jpg)
+
+## Monthly community quiz
+
+Congratulations to [squidstain](https://osu.ppy.sh/users/11073207) for being the first person to answer last month's question and score themselves 1 month of osu!supporter!
+For anyone who missed last month's edition, the question was:
+
+*"What turbulent seas does Bancho sail through in order to deliver your messages?"*
+
+and the answer was: *cho.ppy.sh*. Get it, choppy as in turbulent? As terrible as this pun was, it was enough for squidstain to answer correctly and acquire themselves a free month of osu!supporter. A lot of you had very creative answers and we hope to see that level creativity continue with this month's question, courtesy of our trivia master [chromb](https://osu.ppy.sh/users/10238680) and one of the trivia champs themselves (and previous month's winner), squidstain:
+
+**BYEAH (placeholder)**
+
+When we showed this question to [Ephemeral](https://osu.ppy.sh/users/102335) (the osu! community manager), they liked it so much both due to the difficulty and creativity, that they have decided to grace the first person to correctly answer with **3 months of osu!supporter!**
+
+Same rules apply, you have exactly 1 month after this news post goes up (or until the next news post is up, depending on the timing) to provide a correct answer via a comment, if no correct answer is provided by that time we will reveal the answer in the next edition and the prize will no longer be claimable. Good luck!
+
+## Things to look out for in January
+
+- Yet another LAN tourney is on the horizon. Organized by [Zeus-](https://osu.ppy.sh/users/5464437) with the support of [Boğaziçi University Gaming Club (BUOK)](http://buok.boun.edu.tr/), the very first iteration of **Bosphorus Invitational** aims to gather around some of the best circle clickers within the Turkish community at the border of the magnificent Bosphorus strait where the continents meet, for an epic 1 day showdown of Turkey's best players. While the participation is invite-only, feel free to either visit the venue on the 29th or check out the tournament via the [BUOK Twitch channel](https://www.twitch.tv/buoyunkulubu/) to keep the tabs on. See more on the details through the [forum post](https://osu.ppy.sh/community/forums/topics/1696844), which is also [available in Turkish](https://osu.ppy.sh/community/forums/topics/1696848)!
+- The [New Beginnings Art Contest](https://osu.ppy.sh/home/news/2022-12-24-new-beginnings-voting-open) voting has concluded by the time you are reading this, and should have it's results posted. Go and check out all of the wonderful creations from the community, and if you previously put your votes in, check if the creations you voted for have made their way to the top!
```

Comment:
already [announced on the last day of december](https://osu.ppy.sh/home/news/2022-12-31-new-beginnings-results) :) 

with the dreamxiety section this is also already covered

```suggestion
```